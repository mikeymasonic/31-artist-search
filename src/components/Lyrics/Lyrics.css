@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300&display=swap');

.lyricspre{
  text-align: center;
  margin: 0vw 10vw 0vw 10vw;
  border: .2vw solid rgb(133, 220, 226);
  padding: 1vw;
  border-radius: 1rem;
  font-size: 1vw;
  font-family: 'Montserrat', sans-serif;
}

.lyricsh3 {
  font-size: 2vw;
  color: #f7d89e;
  text-align: center;
  font-weight: 200;
  font-family: 'Montserrat', sans-serif;
}
