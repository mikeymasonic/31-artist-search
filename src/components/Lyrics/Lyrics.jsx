import React from 'react';
import PropTypes from 'prop-types';
import Nav from '../../containers/nav/Nav';

const Lyrics = ({ title, lyrics, artistName }) => {
  return (
    <section>
      <h3>Lyrics for <em>{title}</em> by {artistName}</h3>
      <Nav />
      <pre>{lyrics}</pre>
    </section>
  );
};

Lyrics.propTypes = {
  title: PropTypes.string.isRequired,
  lyrics: PropTypes.string.isRequired,
  artistName: PropTypes.string.isRequired
};

export default Lyrics;
