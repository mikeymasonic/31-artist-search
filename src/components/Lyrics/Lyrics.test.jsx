import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics';

describe('Lyrics component', () => {
  it('renders Lyrics', () => {
    const wrapper = shallow(<Lyrics artistName='Silkworm' title="Never Met A Man I Didn't Like" lyrics="oh man isn't this song great??" />);
    expect(wrapper).toMatchSnapshot();
  });
});
