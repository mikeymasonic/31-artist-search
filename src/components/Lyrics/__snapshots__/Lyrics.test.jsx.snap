// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Lyrics 1`] = `
<section>
  <h3>
    Lyrics for 
    <em>
      Never Met A Man I Didn't Like
    </em>
     by 
    Silkworm
  </h3>
  <Nav />
  <pre>
    oh man isn't this song great??
  </pre>
</section>
`;
